src/compiler/compile/utils/a11y.ts

```diff
@@ -223,3 +224,111 @@ export function is_semantic_role_element(role: ARIARoleDefinitionKey, tag_name:
 	}
 	return false;
 }
+
+// https://html.spec.whatwg.org/multipage/form-control-infrastructure.html#autofilling-form-controls:-the-autocomplete-attribute
+const address_type_tokens = new Set(['shipping', 'billing']);
```

Comment:
I was not able to find any of the following tokens in either `axobject-query` or `aria-query`.